cli/status.go

```diff
@@ -85,12 +89,15 @@ func (status *Status) compile() (string, error) {
 	return bb.String(), nil
 }
 
-func newAPIReachabilityStatus() apiReachabilityStatus {
+func newAPIReachabilityStatus() (apiReachabilityStatus, error) {
+	apiCfg, err := config.NewAPIConfig()
+	if err != nil {
+		return apiReachabilityStatus{}, nil
+	}
 	ar := apiReachabilityStatus{
 		Services: []*apiPing{
 			{Service: "GitHub", URL: "https://api.github.com"},
-			{Service: "Exercism", URL: "http://exercism.io/api/v1"},
-			{Service: "X-API", URL: "http://x.exercism.io"},
+			{Service: "Exercism", URL: apiCfg.BaseURL + apiCfg.Endpoints["ping"]},
```

Comment:
I think this could be `apiCfg.URL("ping")`. It basically does exactly this, so I'm not sure it's an improvement :)